pycparser/c_parser.py

```diff
@@ -920,9 +940,20 @@ def p_specifier_qualifier_list_3(self, p):
     def p_specifier_qualifier_list_4(self, p):
         """ specifier_qualifier_list  : type_qualifier_list type_specifier
         """
-        spec = dict(qual=p[1], storage=[], type=[], function=[])
+        spec = dict(qual=p[1], alignment=[], storage=[], type=[], function=[])
         p[0] = self._add_declaration_specifier(spec, p[2], 'type', append=True)
 
+    def p_specifier_qualifier_list_5(self, p):
+        """ specifier_qualifier_list  : alignment_specifier
+        """
+        spec = dict(qual=[], alignment=[], storage=[], type=[], function=[])
+        p[0] = self._add_declaration_specifier(spec, p[1], 'alignment')
```

Comment:
Same question here, it's not clear why you need to create an empty `spec` and pass it to _add_declaration_specifier

Reply:
Sounds like literal copy-pasting from the existing code, nice catch. Also fixed.